### PR TITLE
LPS-180242 Fix adaptive media test

### PR DESCRIPTION
**Pull request by adolfopa**
# What is this about?

https://issues.liferay.com/browse/LPS-180242

# How do I test it?

Follow the steps described in https://issues.liferay.com/browse/LPS-180242.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
Addressed Bchan's comments: https://github.com/brianchandotcom/liferay-portal/pull/132694#issuecomment-1491865823

File: modules/apps/adaptive-media/adaptive-media-journal-test/src/testIntegration/java/com/liferay/adaptive/media/journal/internal/exportimport/data/handler/test/AMJournalArticleStagedModelDataHandlerTest.java [hunk 1]
```diff
@@ -440,6 +443,24 @@ private String _getStaticContent(FileEntry... fileEntries)
 		return _getContent(sb.toString());
 	}
 
+	private JournalArticle _withPortletImportDisabled(
```
Comment (AliciaGarciaGarcia) [inline modules/apps/adaptive-media/adaptive-media-journal-test/src/testIntegration/java/com/liferay/adaptive/media/journal/internal/exportimport/data/handler/test/AMJournalArticleStagedModelDataHandlerTest.java:446]: I think the test is failing (following the steps on the ticket)

You are now disabling this on the test, so it fails on `if (!ExportImportThreadLocal.isImportInProcess())` 
maybe the test is `_withPortletImportEnabled` (and change to true) but I do not know if this is what you want.

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**